backend/tracim_backend/command/cleanup.py

```diff
@@ -0,0 +1,335 @@
+            "--anonymize-if-needed",
+            help="anonymizes the user where he cannot be deleted",
+            dest="anonymize_if_needed",
+            default=False,
+            action="store_true",
+        )
+        parser.add_argument(
+            "-r",
+            "--delete-all-user-revisions",
+            help="this allow to delete all user revisions. This may create inconsistent database",
+            dest="delete_revisions",
+            default=False,
+            action="store_true",
+        )
+
+        parser.add_argument(
+            "-b",
+            "--best-effort",
+            help="trying doing the best deletion possible, same as -w -a",
+            dest="best_effort",
+            default=False,
+            action="store_true",
+        )
+        parser.add_argument(
+            "-f",
+            "--force",
+            help="force user deletion, same as -r -w. May create inconsistent database",
+            dest="force",
+            default=False,
+            action="store_true",
+        )
+
+        parser.add_argument(
+            "--dry-run",
+            help="dry-run mode",
+            dest="dry_run_mode",
+            default=False,
+            action="store_true",
+        )
+
+        parser.add_argument(
+            "-l", "--login", nargs="+", help="User logins (email)", dest="logins", required=True
+        )
+        parser.add_argument(
+            "--anonymous-name",
+            help="Anonymous user display name to use if anonymize option is activated",
+            dest="anonymous_name",
+            required=False,
+        )
+        return parser
+
+    def take_app_action(self, parsed_args: argparse.Namespace, app_context: AppEnvironment) -> None:
+        self._session = app_context["request"].dbsession
+        self._app_config = app_context["registry"].settings["CFG"]
+
+        delete_user_revision = parsed_args.force or parsed_args.delete_revisions
+        delete_owned_sharespaces = (
+            parsed_args.force or parsed_args.best_effort or parsed_args.delete_sharespaces
+        )
+        anonymize_user_if_needed = parsed_args.best_effort or parsed_args.anonymize_if_needed
+
+        if parsed_args.dry_run_mode:
+            print("(!) Running in dry-run mode, no changes will be applied.")
+        if parsed_args.force:
+            print("(!) Running in force mode")
+        if parsed_args.best_effort:
+            print("(!) Running in best_effort mode")
+
+        if delete_user_revision:
+            print("/!\\ Delete all user revisions, database created may be broken /!\\.")
+        if delete_owned_sharespaces:
+            print("(!) User owned sharespaces will be deleted too.")
+        if anonymize_user_if_needed:
+            print("(!) Will Anonymize user if not possible to delete it")
+            if parsed_args.anonymous_name:
+                print(
+                    '(!) Custom anonymous name choosen is : "{}"'.format(parsed_args.anonymous_name)
```

Comment:
Make verification about wording about anonymous name, user, anonymise, etc....... Like we have in documentation and what is done after translate in french (Wording is it the good one).

Reply:
done.